androidsdk/src/main/res/layout/wootric_survey_layout.xml

```diff
@@ -13,113 +15,144 @@
         android:padding="@dimen/wootric_survey_layout_header_padding"
         android:text="@string/wootric_example_nps_question"/>
 
-    <RelativeLayout
+    <com.wootric.androidsdk.views.phone.ThankYouLayout
+        android:id="@+id/wootric_thank_you_layout"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"/>
+
+    <android.support.constraint.ConstraintLayout
         android:id="@+id/wootric_survey_layout_body"
-        android:focusable="true"
-        android:focusableInTouchMode="true"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:padding="@dimen/wootric_survey_layout_body_padding"
-        android:background="@android:color/white">
+        android:background="@android:color/white"
+        android:focusable="true"
+        android:focusableInTouchMode="true"
+        tools:layout_editor_absoluteY="0dp"
+        tools:layout_editor_absoluteX="0dp">
 
         <LinearLayout
             android:id="@+id/wootric_score_layout"
-            android:layout_width="match_parent"
-            android:layout_height="@dimen/wootric_score_layout_height"
-            android:orientation="horizontal"/>
+            android:layout_width="0dp"
+            android:layout_height="24dp"
+            android:layout_marginLeft="24dp"
```

Comment:
Try setting a variable instead of using `24dp`. So in the dimensions file you could add a variable with this value and then reuse it here.